ontology/yaml/resources/LIGHTING/entity_types/LTGW.yaml

```diff
@@ -14,6 +14,20 @@
 
 # Canonical types for Lighting Gateways
 
+
+LTGW_DFRT_ZLC_LFA:
```

Comment:
Please adjust name to match the total implemented types: `LTGW_LSC_RC_DC_DFRT_ZLC_LFA`

Reply:
oops. done now.